### PR TITLE
Week 2 Announce (typos fixed, banner added)

### PR DESCRIPTION
**Pull request by CSmartt**

### CODE REVIEW

File: source/_posts/2017-Week-2-announce.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Engineering and Computing Ball
+
+The second ever Engineering and Computing Ball will take place on November 16th
+in The Bonnington Hotel (formerly the Regency Hotel). You can sign up today by
+throwing 10 euro at a committee member.  
+https://www.facebook.com/events/312772709198090
+
+
+Your favourite secretary,
```
Comment (butlerx) [inline source/_posts/2017-Week-2-announce.md]: add empty line after this

Comment (CSmartt) [reply source/_posts/2017-Week-2-announce.md]: Doublespaced, should have the same effect.

Comment (butlerx) [reply source/_posts/2017-Week-2-announce.md]: ignore double space

Comment (butlerx) [reply source/_posts/2017-Week-2-announce.md]: in future can we use empty lines so as to make reviewing easier

**Pull request merged**